Formula/geocouch.rb

```diff
@@ -1,11 +1,10 @@
 class Geocouch < Formula
   desc "Spatial index for CouchDB"
   homepage "https://github.com/couchbase/geocouch"
-  url "https://github.com/couchbase/geocouch/archive/couchdb1.3.x.tar.gz"
+  url "https://codeload.github.com/couchbase/geocouch/tar.gz/couchdb1.3.x"
```

Comment:
We should add an `audit` rule to check that these changes (`archive` to `codeload`) aren't made as, although they follow a redirect, GitHub is likely to break them in future.